pom.xml

```diff
@@ -155,6 +161,13 @@
 			<version>2.2.9</version>
 			<scope>test</scope>
 		</dependency>
+
+		<!-- TRUESKILL DEPENDENCIES -->
+		<dependency>
+			<groupId>de.davidbilge</groupId>
+			<artifactId>jskill</artifactId>
+			<version>1.0-SNAPSHOT</version>
```

Comment:
Well,  JSkills is a originally a non-mavenized project:
https://github.com/nsp/JSkills

I found the following mavenized-forks:
https://github.com/davidbilge/JSkills
https://github.com/choweiyuan/JSkills
https://github.com/matthewsford/JSkills
https://github.com/goochjs/JSkills
https://github.com/mzuber/JSkills

All but goochjs's version is marked as snapshots (but i don't think his version is any more stable than the other forks). The drawback of goochjs's version is that it's the only version in its own separate package namespace.

But since the module i included is actually provided by my own github hosted mini maven repository, i could just release the current module as non-snapshot, if that helps in any way? :)